config/default.yml

```diff
@@ -39,7 +39,7 @@ RSpec/DescribedClass:
   - explicit
 
 RSpec/DescribeMethod:
-  Description: Checks that the second argument to `describe` specifies a method.
+  Description: Checks that arguments and sub-describes of `describe` specify a method.
```

Comment:
"Checks that secondary arguments to top level describes and nested describes specify methods